test/common/cdp-driver.js

```diff
@@ -72,6 +73,8 @@ function setupLogging(client) {
         resolveLogPromise();
     });
 
+    client.Runtime.exceptionThrown(info => unhandledExceptions.push(info.exceptionDetails));
```

Comment:
This should also be written to the console. Is it?

Reply:
It could (and indeed I had a local test version that did that). It would replicate the same message as the `testlib.Error`, though, so it wouldn't tell anything new.

Reply:
If there are multiple exceptions, I really want to see them all, not just the last one. Similar to what we do with unexpected journal messages.